esmvalcore/cmor/check.py

```diff
@@ -402,13 +403,27 @@ def _check_time_coord(self):
             self.report_error(self._does_msg, var_name,
                               'have time reference units')
         else:
+            old_units = coord.units
             coord.convert_units(
                 cf_units.Unit(
-                    'days since 1950-1-1 00:00:00',
+                    'days since 1850-1-1 00:00:00',
                     calendar=coord.units.calendar))
             simplified_cal = self._simplify_calendar(coord.units.calendar)
             coord.units = cf_units.Unit(coord.units.origin, simplified_cal)
 
+            attrs = cube.attributes
+            attr = 'branch_time_in_child'
+            if attr in attrs:
+                attrs[attr] = old_units.convert(attrs[attr], coord.units)
+
+            attr = 'parent_time_units'
+            if attr in attrs:
+                parent_units = cf_units.Unit(attrs[attr], simplified_cal)
+                attr = 'branch_time_in_parent'
+                if attr in attrs:
+                    attrs[attr] = parent_units.convert(attrs[attr],
+                                                       coord.units)
```

Comment:
The `parent_time_units` need to be converted to! At the moment the resulting file is wrong.